src/frontend/hooks/useHentFullmektige.ts

```diff
@@ -0,0 +1,33 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useApp } from '../context/AppContext';
+import { FullmektigDto } from '../typer/fullmakt';
+import { IdentRequest } from '../typer/identrequest';
+import { byggTomRessurs, feilmeldingVedFeil, pakkUtHvisSuksess, Ressurs } from '../typer/ressurs';
+
+interface FullmektigeEllerFeilmelding {
+    fullmektige?: FullmektigDto[];
+    hentFullmektigeFeil?: string;
+}
+
+export function useHentFullmektige(fullmaktigiverIdent: string): FullmektigeEllerFeilmelding {
+    const { request } = useApp();
+
+    const [fullmektigeResponse, settFullmektigeResponse] =
+        useState<Ressurs<FullmektigDto[]>>(byggTomRessurs());
+
+    const hentFullmektige = useCallback(() => {
+        request<FullmektigDto[], IdentRequest>(`/api/sak/fullmakt/fullmektige`, 'POST', {
+            ident: fullmaktigiverIdent,
+        }).then(settFullmektigeResponse);
+    }, [fullmaktigiverIdent, request]);
+
+    useEffect(() => {
+        hentFullmektige();
+    }, [hentFullmektige]);
+
+    return {
+        fullmektige: pakkUtHvisSuksess(fullmektigeResponse),
+        hentFullmektigeFeil: feilmeldingVedFeil(fullmektigeResponse),
```

Comment:
Kan denne bare hete `fullmektigFeil` eller `feilmelding` alene? 1) fordi det ikke er en funksjon, som _hent_ gir inntrykk av og 2) hvis ikke den brukes utenfor fullmektige så gir feilmelding alene nok mening.

Innser mens jeg skriver denne at det kanskje skal leses "hent fullmektige" "feil" og ikke "hent" feil for fullmektige, siden hooken heter useHent... Der syns jeg egentlig ikke at hooks trenger å hete `useHent...`, da `useFullmektige` holder i mitt hode men vet ikke hva som er standard 🤷 

Såå evt. forslag er `feilmeldingFraHenting` eller `feilmeldingFraHentFullmektig` for å vise at det er en variabel og ikke en funksjon

Reply:
Ingenting av dette er vel egentlig standard uansett. Mer vanlig er å bruke et rammeverk som _SWR_ eller _TanStack Query_, og da slippe unna å skrive egne fetch-hooks

`feilmelding` er jeg negativ til å kalle den, fordi dersom du bruker denne hooken i en større komponent så må du uansett rename det til noe som gir mer konstekst.

`feilmeldingFraHentFullmektig` er et bra forslag